ACHNBrowserUI/ACHNBrowserUI/views/design/DesignListView.swift

```diff
@@ -7,15 +7,20 @@
 //
 
 import SwiftUI
-import Backend
 
 struct DesignListView: View {
 
     // MARK: - Properties
 
-    @EnvironmentObject private var collection: UserCollection
+    @ObservedObject private var viewModel: DesignListViewModel
```

Comment:
@Dimillian, @TheVaan, I think that can help us out to do the same thing in `CustomTasksListView`. Using a `ViewModel` instead of direct access to the Singleton is better IMO.

Reply:
It depend because in the end it’s doing exactly the same thing -> update your view when the envObject publish its properties. You’ll subscribe to the collection property anyway in the view model.

Reply:
But yeah yo can scope the update and subscribe to only what matter to you. But UserCollection is only publishing its various lists, and the way it is right now, you can’t really receive multiple update at once as the user can only do one action simultaneously. Anyway I’ll try when I’ll take a look tomorrow.